src/action/PythonAction.py

```diff
@@ -1,9 +1,9 @@
 mySourceData = 0
 myTargetData = 0
 
-def performAction(time, dt, sourceData, targetData):
+def performAction(time, timeWindowSize, sourceData, targetData):
```

Comment:
Do we really want the time window size here and not the timestep size?

Reply:
I think this is not really defined yet. I just decided to use the `timeWindowSize`.

Reply:
@fsimonis any opinion? From my point of view we just have to decide on A or B here, but I might be missing something. I decided for `timeWindowSize`.

Reply:
In https://github.com/precice/precice/pull/935/files/0ee5d5af9f5a1393c49812dd6869808a45ce6aeb#diff-b613b1ccaeb22bcb08750b2aa4fa92acb0fa8a1bf151cab3043dd2b2f77319e0L6 it says `(global) timestep size`. I translated this to `timeWindowSize`. So everything is fine.

Reply:
My gut feeling tells me that what we want depends on when the action is called.
I guess that the timestep size make sense for actions based on mappings and that the timewindow size makes sense for an action based on TIME_WINDOW_COMPLETE.

Not changing the behaviour is probably the best here.
We could also pass both by adding another callback in another PR.